tests/jest/plugins/search/utils.test.js

```diff
@@ -0,0 +1,18 @@
+import { marshallSearchResults } from '@/src/plugins/search/utils.js';
+import { deepCopy } from '@/tests/jest/utils.js';
+import { DUMMY_RESULTS } from './utils.js';
+
+describe('marshallSearchResults', () => {
+  test('Adds match index', () => {
+    const results = deepCopy(DUMMY_RESULTS);
+    marshallSearchResults(results, x => x.toString(), '{{{', '}}}');
```

Comment:
q: do we need to show new delimiter characters as example?

Reply:
Great idea! I added a test case to this file with the "official" delimiters.